dmutils/email.py

```diff
@@ -231,3 +232,95 @@ def decode_invitation_token(encoded_token):
 
 def token_created_before_password_last_changed(token_timestamp, user_timestamp):
     return token_timestamp < user_timestamp
+
+
+class DMNotifyClient(object):
+    """Digital Marketplace wrapper around the Notify python client."""
+
+    client_class = NotificationsAPIClient
```

Comment:
Fair enough. I somehow missed the fact that it's a class attribute, I got the impression that this was a global variable. Let's make it `_client_class` though, to make it clear that it's not supposed to be used externally.